examples/blog/stub-server.json

```diff
@@ -80,8 +80,9 @@
       "teaser": "Consequuntur id aut soluta aspernatur sit. Aut doloremque recusandae sit saepe ut quas earum. Quae pariatur iure et ducimus non. Cupiditate dolorem itaque in sit.",
       "body": "<p>Aut molestiae quae explicabo voluptas. Assumenda ea ipsam quia. Rerum rerum magnam sunt doloremque dolorem nulla. Eveniet ut aliquam est dignissimos nisi molestias dicta. Dolorum et id esse illum. Ea omnis nesciunt tempore et aut. Ut ullam totam doloribus recusandae est natus voluptatum officiis. Ea quam eos velit ipsam non accusamus praesentium.</p><p>Animi et minima alias sint. Reiciendis qui ipsam autem fugit consequuntur veniam. Vel cupiditate voluptas enim dolore cum ad. Ut iusto eius et.</p><p>Quis praesentium aut aut aut voluptas et. Quam laudantium at laudantium amet. Earum quidem eos earum quaerat nihil libero quia sed.</p><p>Autem voluptatem nostrum ullam numquam quis. Et aut unde nesciunt officiis nam eos ut distinctio. Animi est explicabo voluptas officia quos necessitatibus. Omnis debitis unde et qui rerum. Nisi repudiandae autem mollitia dolorum veritatis aut. Rem temporibus labore repellendus enim consequuntur dicta autem. Illum illo inventore possimus officiis quidem.</p><p>Ullam accusantium eaque perspiciatis. Quidem dolor minus aut quidem. Praesentium earum beatae eos eligendi nostrum. Dolor nam quo aut.</p><p>Accusamus aut tempora omnis magni sit quos eos aut. Vitae ut inventore facere neque rerum. Qui esse rem cupiditate sit.</p><p>Est minus odio sint reprehenderit. Consectetur dolores eligendi et quaerat sint vel magni. Voluptatum hic cum placeat ad ea reiciendis laborum et. Eos ab id suscipit.</p><p>Enim corporis tenetur quia molestias qui tempore ullam. Explicabo sunt quis alias autem tempora modi aliquid. Modi in enim neque enim. Natus quia dolor eum. Deserunt porro ex doloribus quis architecto impedit soluta. Accusamus sunt animi quidem expedita ullam reiciendis vitae. Autem voluptates neque rerum amet. Vero non ipsa veritatis aspernatur placeat. Pariatur sequi voluptas eos qui pariatur dignissimos aut et. Ut odit et ipsa provident eum. Omnis autem eveniet expedita maiores repellendus quia. Et voluptas quo eum exercitationem impedit vitae.</p>",
       "views": 559,
-      "published_at": "2012-08-24",
-      "tags": []
+      "published_at": "2012-08-22",
```

Comment:
Should revert this file, and perhaps try to ignore further modifications on it using `.gitignore`?